README.md

```
@@ -146,6 +146,12 @@ You can find the API reference [here](https://mailtrap.github.io/mailtrap-java/i
 
 Bug reports and pull requests are welcome on [GitHub](https://github.com/mailtrap/mailtrap-java). This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [code of conduct](CODE_OF_CONDUCT.md).
 
+### Build
+- Istall [direnv](https://direnv.net/)
+- Install [asdf](https://asdf-vm.com/)
+- Install Java plugin https://github.com/halcyon/asdf-java
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)